resources/zib/zib-Procedure.xml

```diff
@@ -14,11 +14,12 @@
       <use value="work" />
     </telecom>
   </contact>
-  <purpose value="This Procedure resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) ]()." />
+  <description value="This profile covers the recording of therapeutic or diagnostic procedure undergone by the patient. The ServiceRequest profile (http://nictiz.nl/fhir/StructureDefinition/zib-Procedure-ServiceRequest) covers the procedure that the patient will undergo." />
+  <purpose value="This Procedure resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Procedure](https://zibs.nl/wiki/Procedure-v5.2(2020EN))." />

```

Comment:
Not conform the profiling guidelines:

This [resource type] resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) [English zib name] [version]([release])]([link to the English zib page on zibs.nl]).

Reply:
Fixed!